### DISCUSSION TITLE
FR: Select slot to save the UBL mesh from the menu

### DISCUSSION THREAD

**Discussion by kted** [Jyers/Marlin#488] [Ideas]
First off, thanks for a great firmware!
Now for a feature request: The firmware has the capability to store 2 UBL meshes. By default, it stores the mesh in slot 0. To store it in slot 1, you have to use a terminal, and issue g-code commands. I use both slots, one for the textured, and one for the smooth side of the glass bed, and they are different, since it is bulging a bit in the middle.
Can you make the UBL slot to save the mesh generated by using the menu, selectable? 
I mostly operate the printer remotely, and there are non-technical people involved on the other end, so making this accessible from the menu would be a great help.

**Comment by Jyers** [comment]
I was actually planning on making this, but ran into the issue that the 15x15 UBL (which I usually run) only has enough eeprom space for one mesh. But if 10x10 has room for two I will definitely add this to the firmware!
